core/src/main/java/com/yahoo/oak/OakHashMap.java

```diff
@@ -54,9 +54,16 @@
         this.entryDeserializeTransformer = entry -> new AbstractMap.SimpleEntry<>(
                 keySerializer.deserialize(entry.getKey()),
                 valueSerializer.deserialize(entry.getValue()));
-        this.internalOakMap = new InternalOakMap<>(minKey, keySerializer, valueSerializer, oakComparator,
-                this.valuesMemoryManager, kMM, chunkMaxItems, new ValueUtils());
 
+        // In order to use USE_DEFAULT_FIRST_TO_SECOND_BITS_PARTITION configuration
+        // we need to let Java to use about 14GB of onheap memory anywhere OakHashMap is used,
+        // also for testings (each test allocate and release!!!).
+        // Therefore using less here: 2^log2NumOfChunks <-- number of chunks;
+        // 2^(log2NumOfChunks*2) <-- number of entries in each chunk
```

Comment:
There is no big logic here, I just tried to make the tests working without hitting Out-Of-Memory exception due to to big memory requirement. I still wanted it to be more or less common, without each test requesting specific number of memory to be used. This is the numbers where it worked. Of course it needs to be changed when going to commit/release. I added TODO comment.

Reply:
Why are you hitting OOM? OakHashMap builder should have a `maxItems` parameter so we can set it to a lower number for the tests.

Reply:
I was hitting OOM due to too big static memory requests in the tests. On the other way, if I allocate too little memory (few small chunks) I am way too many collisions, which I want to avoid as well. 

OakBuilder does have `maxItemsPerChunk` but it can not be translated the same to OakHash. In OakMap this number means number of items in one chunk and there going to be more if this is not enough. But in OakHash this gives a hint how much items in every chunk and how much of them to preallocate.

Anyway, this is going to be changed soon.